code/modules/jobs/job_types/roguetown/yeomen/archivist.dm

```diff
@@ -49,7 +49,7 @@
 	)
 
 	subclass_stats = list(
-		STATKEY_INT = 4,
+		STATKEY_INT = 6, //The Archivist must always be the smartest guy in the realm, this is essential for the sovl. Duke has +5.
```

Comment:
I forgot I did this, +6 is sortof absurd. Should I undo this? Nerfing Heavy Knight and Merchant Lord to not be supergeniuses is out of scope, would need to be done in another PR.